classes/class-twenty-twenty-one-custom-colors.php

```diff
@@ -64,15 +64,16 @@ public function custom_get_readable_color( $background_color ) {
 	public function generate_custom_color_variables( $context = null ) {
 
 		$theme_css = 'editor' === $context ? ':root .editor-styles-wrapper{' : ':root{';
```

Comment:
Coding Standards: Align `=` signs.
```suggestion
		$theme_css        = 'editor' === $context ? ':root .editor-styles-wrapper{' : ':root{';
```